src/events.rs

```diff
@@ -37,12 +37,21 @@ pub enum WindowEvent {
     Destroyed,
 
     /// A file has been dropped into the window.
+    /// 
+    /// When the user drops multiple files at once, this event will be emitted for each file
+    /// separately
```

Comment:
```suggestion
    /// separately.
```
Could you please add a period? After that, we'll be good to merge!